src/_overrides.scss

```diff
@@ -57,3 +57,7 @@
     transition: 0s;
   }
 }
+
+.level-left {
+  display: flex;
+}
```

Comment:
Please don't override a bulma layout. This will cause weirdness if we try and use it in other places